.github/workflows/checkAndSubmitAddonMetadata.yml

```diff
@@ -219,44 +224,24 @@ jobs:
         # If no valid result is produced, i.e. no add-on id is found, jq exits with status 4.
         # If there is a different error, a different non-zero exit code is used.
         exit $? # Exit with the same exit code as jq
-
+        continue-on-error: true
     - name: Create discussion for add-on id
       id: createDiscussion
-      if: ${{ steps.checkDiscussion.outcome }} == 'failure'
+      if: (${{ success() }} || ${{ failure() }}) && (${{ steps.checkDiscussion.outcome }} == 'failure' || 
       uses: abirismyname/create-discussion@v1.1.0
       env:
         GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}      
       with:
-        repository-id: $GITHUB_REPOSITORY_ID
+        repository-id: 'MDEwOlJlcG9zaXRvcnkyMzI4MTU5NTQ='
         category-id: 'DIC_kwDODeB9Us4CaB4B'
         title: Reviews for ${{ steps.getAddonNameAndVersion.outputs.addonName }} add-on
         body: |
-          Community add-on reviews for {{ steps.getAddonNameAndVersion.outputs.addonName }}.
+          Community add-on reviews for ${{ steps.getAddonNameAndVersion.outputs.addonName }}.
           Please use threaded replies and avoid email responses.
           Reviews for specific versions of the add-on can be added in response to automated comments for each version.
           Reviews should abide by the [NVDA code of conduct](https://github.com/nvaccess/nvda/blob/master/CODE_OF_CONDUCT.md).
-    - name: Create review comment
-      if: ${{ steps.checkDiscussion.outcome }} == 'success'
-      uses: actions/github-script@v6
-      with:
-        script: |
-          const fs = require('fs')
-          const contents = fs.readFileSync('discussions.json')
-          const data = JSON.parse(contents)
-          const addonName = "${{ steps.getAddonNameAndVersion.outputs.addonId }}"
-          const addonVersion = "${{ steps.getAddonNameAndVersion.outputs.addonVersion }}"
-          const discussionId = data[addonId]["discussionId"]
-          const mutation = `mutation {
-            addDiscussionComment(input: {body: "Reviews for version ${addonVersion}", discussionId: "${discussionId}", clientMutationId: "GitHub Script action"}) {
-              clientMutationId
-              comment {
-                body
-            }
-            }
-          }`;
-          github.graphql(mutation)
     - name: Add add-on ID and discussion data to json file
-      if: ${{ steps.checkDiscussion.outcome }} == 'failure'
+      if: (${{ success() }} || ${{ failure() }}) && (${{ steps.checkDiscussion.outcome }} == 'failure'
```

Comment:
same here. this is missing a `)` at the end. Do we really need to to check both success and failure? Isn't this the same as `always()`?